### PR TITLE
Throw if a run-remote-script action is missing an entry_point parameter

### PR DESCRIPTION
**Pull request by Kami**
run-remote-script actions without entry_point don't make sense.

This would save @jfryman and I at least 30 minutes :P

Before:

```bash
id: 5500c10d0640fd0d53077989
status: failed
result: 
{
    "xxx": {
        "failed": true, 
        "traceback": "  File "/data/stanley/st2common/st2common/models/system/action.py", line 395, in _sudo
    'return_code': output.return_code,
  File "/data/stanley/virtualenv/local/lib/python2.7/site-packages/fabric/network.py", line 639, in host_prompting_wrapper
    return func(*args, **kwargs)
  File "/data/stanley/virtualenv/local/lib/python2.7/site-packages/fabric/operations.py", line 1095, in sudo
    stderr=stderr, timeout=timeout, shell_escape=shell_escape,
  File "/data/stanley/virtualenv/local/lib/python2.7/site-packages/fabric/operations.py", line 895, in _run_command
    _prefix_commands(_prefix_env_vars(command), 'remote'),
  File "/data/stanley/virtualenv/local/lib/python2.7/site-packages/fabric/operations.py", line 704, in _prefix_env_vars
    return shell_env_str + command
", 
        "succeeded": false, 
        "error": "coercing to Unicode: need string or buffer, NoneType found"
    }
}
```

After:

```bash
id: 5500c48a0640fd0d530779b6
status: failed
result: Message: Action "apt_get_update" is missing entry_point attribute. Perhaps wanted to use "run-remote" runner?
Traceback:   File "/data/stanley/st2actions/st2actions/container/base.py", line 103, in _do_run
    (status, result, context) = runner.run(action_params)
  File "/data/stanley/st2actions/st2actions/runners/fabricrunner.py", line 121, in run
    remote_action = self._get_fabric_remote_script_action(action_parameters)
  File "/data/stanley/st2actions/st2actions/runners/fabricrunner.py", line 170, in _get_fabric_remote_script_action
    raise Exception(msg % (self.action_name))
```

### CODE REVIEW

File: st2actions/st2actions/runners/fabricrunner.py [hunk 1]
```diff
@@ -114,9 +114,16 @@ def pre_run(self):
 
     def run(self, action_parameters):
         LOG.debug('    action_parameters = %s', action_parameters)
-        remote_action = self._get_fabric_remote_action(action_parameters) \
```
Comment (Kami) [inline st2actions/st2actions/runners/fabricrunner.py]: This was just a code smell, we should select a method based on the runner type and not the presence or value of the entry_point.

Comment (lakshmi-kannan) [inline st2actions/st2actions/runners/fabricrunner.py]: Fine.

**Comment by lakshmi-kannan** [comment]
>Perhaps wanted to use "run-remote" runner?
"Perhaps you want "run-remote" instead?" might be better?

**Comment by lakshmi-kannan** [comment]
LGTM. Thanks for picking this up.

**Comment by Kami** [comment]
Re message - Yeah, usually I think additional statement like this belong to the CLI / webui, but I think returning it directly from the API here makes sense since it's a mistake I imagine a lot of people could make.

**Pull request merged**